src/Microsoft.AspNetCore.Rewrite/Internal/UrlActions/ChangeEnvironmentPreAction.cs

```diff
@@ -4,17 +4,17 @@
 using System;
 using Microsoft.AspNetCore.Http;
 
-namespace Microsoft.AspNetCore.Rewrite.Internal.PreActions
+namespace Microsoft.AspNetCore.Rewrite.Internal.UrlActions
 {
-    public class ChangeEnvironmentPreAction : PreAction
+    public class ChangeEnvironmentPreAction : UrlAction
```

Comment:
Can you also rename the class "ChangeEnvironmentPreAction" => "ChangeEnvironmentAction"?